test/e2e/egress_test.go

```diff
@@ -717,21 +719,18 @@ func (data *TestData) createEgress(t *testing.T, generateName string, matchExpre
 }
 
 func (data *TestData) waitForEgressRealized(egress *v1alpha2.Egress) (*v1alpha2.Egress, error) {
-	err := wait.PollImmediate(200*time.Millisecond, 5*time.Second, func() (done bool, err error) {
+	err := wait.PollImmediate(200*time.Millisecond, waitNodeFailureTimeout, func() (done bool, err error) {
```

Comment:
It doesn't look suitable to use waitNodeFailureTimeout for waiting for egress realized

Reply:
Done
Has been updated to `waitEgressRealizedTimeout`.